packages/expo-notifications/src/NotificationsEmitter.ts

```diff
@@ -129,10 +129,21 @@ export function removeNotificationSubscription(subscription: EventSubscription)
  * - a [`NotificationResponse`](#notificationresponse) object - if a notification response was received
  */
 export async function getLastNotificationResponseAsync(): Promise<NotificationResponse | null> {
-  if (!NotificationsEmitterModule.getLastNotificationResponseAsync) {
-    throw new UnavailabilityError('ExpoNotifications', 'getLastNotificationResponseAsync');
+  return getLastNotificationResponse();
+}
+
+/**
+ * Gets the notification response that was received most recently
+ * (a notification response designates an interaction with a notification, such as tapping on it).
+ *
+ * - `null` - if no notification response has been received yet
+ * - a [`NotificationResponse`](#notificationresponse) object - if a notification response was received
+ */
+export function getLastNotificationResponse(): NotificationResponse | null {
+  if (!NotificationsEmitterModule.getLastNotificationResponse) {
+    throw new UnavailabilityError('ExpoNotifications', 'getLastNotificationResponse');
   }
```

Comment:
We need to keep this code that checks for the availability of the native API, but have it check for `getLastNotificationResponse`.

Reply:
The code that checks for the availability of the native API is still there. I moved it from async functions to sync functions as the async functions call sync ones anyway. See lines 143-144.